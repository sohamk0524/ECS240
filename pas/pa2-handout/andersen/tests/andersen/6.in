c Test case with address-of and multiple derefs on LHS and RHS
p 7 8
s 0 1 -1 2
s 0 2 -1 2
s 0 2 -1 3
s 0 3 -1 4
s 0 4 -1 5
s 0 6 -1 7
s 2 1 10 4
s 3 1 -1 6
