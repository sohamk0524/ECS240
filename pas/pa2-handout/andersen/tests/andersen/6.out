pt 1 2
pt 2 2
pt 2 3
pt 2 6
pt 2 7
pt 3 4
pt 3 6
pt 3 7
pt 4 5
pt 4 6
pt 6 6
pt 6 7
pt 7 6
pt 7 7
